### DISCUSSION TITLE
v1.6.0 is released

### DISCUSSION THREAD

**Discussion by mikekazakov** [mikekazakov/nimble-commander#284] [Announcements]
What's new in the version:
  - Nimble Commander is now completely free and open source.
  - Added support for file tags.
  - Fixed an issue that could stop directory updates on SMB volumes.
  - Fixed an issue where the mouse cursor could navigate to the wrong directory.
  - You can now extract files from archives by dragging them into the ".." directory.
  - Fixed an issue where file permissions might not be restored correctly after extracting from an archive.
  - Fixed a crash when accessing an FTP server.
  - Fixed keyboard navigation issues with F1/F2 and Quick Lists pop-ups.
  - Fixed a bug preventing correct duplication of a file that's being written to.

**Comment by maxim-uvarov** [comment]
@mikekazakov Thank you so much for developing and sharing such a powerful tool!
It's the very best file manager for Mac!

**Comment by pretzelmonster** [comment]
Yes, thanks @mikekazakov for the continous development and good decision to make it open source now. I have been happily paying and using it for 3 years now after extensive market research and it is the best on MacOS!
Keep up the good work and maybe you even get some more support and visibility now!

**Comment by postacik** [comment]
I've recently found this application. It's really perfect for developers. Thanks for making it free and open source.

**Reply by erictioh** [reply]
i love the command line directly in sync with the GUI navigation, anything that the GUI is not capable / not suitable of, the command line come into direct compliment immediately. 

for e.g. i want my current full path, i will just do `pwd`. or i want to extract the list of file with specific keyword, i will just `ls | grep abc`

**Comment by meglio** [comment]
Hope making it free won't make you less passionate about the product long term. I'm a paid customer and I thank you for making it available for more people.

**Reply by mikekazakov** [reply]
You are welcome! I hope that as well; my interest in developing Nimble Commander isn't going anywhere. Though what has changed after fully opening is that a default response to feature requests became "pull requests are welcome".
